src/pudl/transform/params/ferc1.py

```diff
@@ -3262,6 +3262,166 @@
+                        "amortization_of_loss_on_reacquired_debt",
+                        "amortization_of_gain_on_reacquired_debt_credit",
+                        "loss_on_disposition_of_property",
+                        "taxes_other_than_income_taxes_other_income_and_deductions",
+                        "life_insurance",
+                        "income_before_extraordinary_items",
+                        "net_interest_charges",
+                        "maintenance_expense",
+                        "revenues_from_merchandising_jobbing_and_contract_work",
+                        "other_deductions",
+                        "operating_revenues",
+                        "provision_for_deferred_income_taxes_credit_other_income_and_deductions",
+                        "extraordinary_items_after_taxes",
+                        "revenues_from_nonutility_operations",
+                        "taxes_other_than_income_taxes_utility_operating_income",
+                        "utility_operating_expenses",
+                        "gain_on_disposition_of_property",
+                        "extraordinary_deductions",
+                        "interest_and_dividend_income",
+                        "income_taxes_federal",
+                        "extraordinary_income",
+                        "amortization_of_premium_on_debt_credit",
+                    ]
+                }
+            },
+        },
+        "wide_to_tidy": {
+            "xbrl": {
+                "idx_cols": ["entity_id", "report_year", "utility_type_axis"],
+                "value_types": ["income"],
+                "expected_drop_cols": 2,
+                "stacked_column_name": "xbrl_factoid",
+            },
+            "dbf": {
+                "idx_cols": [
+                    "utility_id_ferc1",
+                    "report_year",
+                    "income_type",
+                    "record_id",
+                ],
+                "value_types": ["income"],
+                "expected_drop_cols": 12,
+                "stacked_column_name": "utility_type",
+            },
+        },
+        "drop_duplicate_rows_dbf": {
+            "data_columns": [
+                "income",
+            ],
+            "table_name": "income_statement_ferc1",
+        },
+        "align_row_numbers_dbf": {
+            "dbf_table_names": ["f1_income_stmnt", "f1_incm_stmnt_2"]
+        },
+        "categorize_strings": {
+            "utility_type": {
+                "categories": {
+                    "total": ["total"],
+                    "electric": ["electric", "ferc:ElectricUtilityMember"],
+                    "gas": ["gas", "ferc:GasUtilityMember"],
+                    "common": ["common", "ferc:CommonUtilityMember"],
+                    "other1": ["other1", "ferc:OtherUtilityMember"],
+                    "other2": ["other2"],
+                    "other3": ["other3"],
+                    "other4": ["other4"],
```

Comment:
I edited the categorization so that the xbrl `ferc:OtherUtilityMember` and dbf `other_total` are both categorized as `other`. the dbf `other{n}` values are additional details within the other category that does not exist in the xbrl data. I believe this categorization will enable us to use the xbrl calculations seamlessly - the dbf `other{n}` categories will not show up in the xbrl calcs and that is okay for aggregations because `other` should be the total of all of the `other{n}` values - an assumption we should test in the aggregation stage.